packages/origin-ui-core/src/components/Organization/DownloadDocuments.tsx

```diff
@@ -17,7 +17,7 @@ export const downloadFile = async (client: FileClient, id) => {
             const url = window.URL.createObjectURL(blob);
             const link = document.createElement('a');
             link.href = url;
-            link.setAttribute('download', name);
+            link.setAttribute('download', 'name');
```

Comment:
@alexworker23 I'm not what should be instead of `name` here, but since now `name` variable is depreciated. Please check and change if necessary.